### ISSUE TITLE
importFromIMGT on Windows

### ISSUE THREAD

**Issue by SolaimanR** [milaboratory/mixcr#125]
Hey,

is there any chance to run the importFromIMGT script from windows? Using `mixcr importFromIMGT` gives `Error: Expected a command, got importFromIMGT`.

Probably difficult, as it is a shell script? I tried to do it in Linux and copy and paste all of the downloaded content on my windows machine. However, it says `Error: Segment library <local> not found.`

**Comment by dbolotin** [comment]
Hi, unfortunately, current version does not support import from IMGT on Windows. Good news is that, after 2.0 release there will be no more need for this action.

We just developed new segment library format and created open repository for known segments.

See [here](https://github.com/repseqio/repseqio) for software to work with format, and [here](https://github.com/repseqio/library) for the library.

We plan to convert segments from IMGT to this format and upload them to the same or separate GitHub repo, so it will be possible to just download and use the library as:
```
mixcr align --library imgt.json ...
```

The other thing is that all minor issues with our [built-in library](https://github.com/repseqio/library) will be fixed by 2.0 .

BTW, it would be interesting to know the reason you want to use IMGT library instead of built-in?

**Comment by SolaimanR** [comment]
Hey,

im looking foward to version 2.0. The reason we would like to use IMGT is
because we have used it for all other analysis for data we are about to
publish. We did not want to change methods for revision work. But i
noticed, there is still some difference when aligning/annotating using
mixcr. Im guessing the alogrythim does differ in the default settings. But
still, great work from your lab, I have based our pipeline on VDJtools, tcR
package and now mixcr. In addition to some custom scripts we have a solid
pipeline. Thank you.

2016-08-19 12:59 GMT+02:00 Dmitry Bolotin <notifications@github.com>:
